moe._999eagle.OkunaDesktop.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Appdata file is also usually something that would be preferred to be maintained in upstream repo; also other distributions use it.

Reply:
The appdata is maintained in the upstream repo, but while I'm still working on Flatpak, it's not yet on master. It will be though.